tests/behat/features/asp/homepage/navigation_bar_footer.feature

```diff
@@ -7,6 +7,26 @@ Feature: test Navigation Bar footer
   Verify NSF acknowledgement and TODO: social links
   Verify acceptable use, code of conduct, and privacy policy
 
+
+  # The following scenario passes, but mysteriously, causes the next scenario to fail.  Disabling for now.
+  #Scenario: Unauthenticated user tests Q&A Forum
+    Given I am not logged in
+    When I go to the homepage
+    #And I wait 4 seconds
+    When I follow "Q&A Forum"
+    #And I wait 4 seconds
+    Then I should be on "https://ask.cyberinfrastructure.org/"
+    #And I wait 4 seconds
+    Given the cache has been cleared
```

Comment:
Maybe the cache clear is causing the problem?

Reply:
with or without, still had the problem.  per our slack, i've simplified all of this to just check that the link points to the right place.